### ISSUE TITLE
Load generator instance was not created

### ISSUE THREAD

**Issue by yuriatgoogle** [GoogleCloudPlatform/stackdriver-sandbox#37]
I just created a new Google account (stackdoctor@gmail.com) to create a new sandbox.  Here's the last part of Cloud Shell output.  It did not create the load generator instance.

********************************************************************************
Stackdriver Sandbox deployed successfully!
     Stackdriver Dashboard: https://app.google.stackdriver.com/accounts/create
     Google Cloud Console Dashboard: https://console.cloud.google.com/kubernetes/workload?project=stackdriver-sandbox-347738008
     Hipstershop web app address: http://34.67.163.61
     Load generator web interface: [not found]
********************************************************************************

**Comment by yuriatgoogle** [comment]
Full output:

```
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
waiting for load generator instance...
error: load generator unreachable
********************************************************************************
Stackdriver Sandbox deployed successfully!
     Stackdriver Dashboard: https://app.google.stackdriver.com/accounts/create
     Google Cloud Console Dashboard: https://console.cloud.google.com/kubernetes/workload?project=stackdriver-sandbox-347738008
     Hipstershop web app address: http://34.67.163.61
     Load generator web interface: [not found]
********************************************************************************
```

**Comment by yuriatgoogle** [comment]
![image](https://user-images.githubusercontent.com/26609620/67419699-092cdf00-f582-11e9-8910-4e2dbb92b4f0.png)
No logs created for the load gen VM...

**Comment by eaberman** [comment]
This is an issue I had when setting up my Sandbox as well.

Debugging: I tried running the scripts ./delete.sh and ./install.sh  several times to get new versions of the Sandbox instance. Eventually, I was able to get the load generator up, but I'm not sure what made that time different than the rest, although it was a few hours after I had initially started trying this.

Screenshots are attached with the error message and the successful instance creation.
![Screenshot 2020-06-30 at 12 28 12 PM](https://user-images.githubusercontent.com/38052620/86616339-4fdd3080-bf7b-11ea-858d-7f34d435d01d.png)

![Screenshot 2020-06-30 at 6 00 53 PM](https://user-images.githubusercontent.com/38052620/86616430-700cef80-bf7b-11ea-8810-1eda78dce9e1.png)

**Comment by simonz130** [comment]
@gzhao408, this bug is FYI. Levy will be working on improving provisioning story and since this is related to Loadgen, you might need to get involved here as well.

**Comment by nicklin37** [comment]
I believe this is fixed in #139 

We added some work during the config to wait for the services to become available before going into the loadgenerator setup https://github.com/GoogleCloudPlatform/stackdriver-sandbox/pull/139#issuecomment-662769060

This fixed the loadgenerator issues. 

@Daniel-Sanche please confirm and if so then I think we can close this issue.

**Comment by Daniel-Sanche** [comment]
Yeah, I think it's safe to close this now. We made sure the Loadgenerator was set up as part of our v0.2.0 launch
